source/backups.rst

```diff
@@ -221,8 +223,46 @@ Following steps are needed to restore a previously saved backup:
 
       kubectl get pxc
 
-3. When both correct names are known, it is needed to fix ``spec.pxcCluster`` and ``spec. backupName`` fields in ``deploy/backup/restore.yaml`` file. After that, the actual restoration process can
-   be started as follows:
+   .. note:: Obviously, you can make this checks only on the same cluster on
+      which you have previously made the backup.
+
+3. When both correct names are known, it is needed to set appropriate keys
+   in the ``deploy/backup/restore.yaml`` file.
+
+   * set ``spec.pxcCluster`` key to the name of your cluster,
+   * set ``spec.backupName`` key to the name of your backup,
+   * if you are restoring backup on the Kubernetes-based cluster different from
+     one you have used to save this backup, set ``spec.backupSource`` subsection
+     to point on the appropriate PVC or S3-compatible storage:
+
+     A. If backup was stored on the PVC volume, ``backupSource`` should contain
+        the PVC-based storage path and name:
+
+        .. code-block:: yaml
+
+           ...
+           backupSource:
+             destination: pvc/PATH_TO_PVC_VOLUME
+             storageName: pvc
+             ...
+
+     B. If backup was stored on the S3-compatible storage, ``backupSource``
+        should contain ``destination`` key equal to the s3 bucket with a special
+        ``s3://`` prefix, followed by the necessary S3 configuration keys, same
+        as in ``deploy/cr.yaml`` file:
+
+        .. code-block:: yaml
+
+           ...
+           backupSource:
+             destination: s3://S3-BUCKET-NAME
```

Comment:
```suggestion
             destination: s3://S3-BUCKET-NAME/BACKUP-NAME
```
@qJkee what do you think?

Reply:
Should we add this?
Since we are passing it to xtrabackup as `S3_BUCKET_URL` env